### ISSUE TITLE
Recurring Deposit Application Missing 

### ISSUE THREAD

**Issue by francisguchie** [openMF/android-client#1774]
**Summary:** 

As a field staff, I wish to be able to apply for a new Recurring Deposit Account, just like one can apply for a new loan or savings account

**Steps to reproduce:** 
login into the android client, select a client of your choice, click on the hamburger icon on the top right corner, you will not see Add Recurring Deposit Account

**Expected behaviour:** 
one should see "Add Recurring Deposit Account" 

**Device and Android version:** 

Device - Tecno Camon 16 Premier, Android 10, 8gb RAM, 128gb ROM

**Comment by francisguchie** [comment]
@danishjamal104 I would like to hear your comments

**Comment by danishjamal104** [comment]
Hey @francisguchie, how about not giving an option for adding `Recurring Deposit Account` but adding the feature to make account's current account a recurring account.?

**Comment by francisguchie** [comment]
@danishjamal104  
There are 4 product types available in FINERACT that need to be applied for differently since they take different forms
1-Loans
2- Savings
3- Fixed Deposits
4- Recurring Deposits 

at the moment the app has 1 and 2 only available at the Client Drop Down (hamburger menu). So no need to change what is available. We need to add the 2 missing

**Comment by danishjamal104** [comment]
> @danishjamal104
> There are 4 product types available in FINERACT that need to be applied for differently since they take different forms
> 1-Loans
> 2- Savings
> 3- Fixed Deposits
> 4- Recurring Deposits
> 
> at the moment the app has 1 and 2 only available at the Client Drop Down (hamburger menu). So no need to change what is available. We need to add the 2 missing

Fine, then we can go ahead. what do you say @robustTechie

**Comment by robustTechie** [comment]
> > @danishjamal104
> > There are 4 product types available in FINERACT that need to be applied for differently since they take different forms
> > 1-Loans
> > 2- Savings
> > 3- Fixed Deposits
> > 4- Recurring Deposits
> > at the moment the app has 1 and 2 only available at the Client Drop Down (hamburger menu). So no need to change what is available. We need to add the 2 missing
> 
> Fine, then we can go ahead. what do you say @robustTechie ?

Yeah, this looks good to me we can add the recurring and fixed deposit feature too in the app.

**Comment by francisguchie** [comment]
@danishjamal104  
would you like to help?

**Comment by therajanmaurya** [comment]
Moved on Jira https://mifosforge.jira.com/browse/MIFOSAC-357
